lib/l10n/intl_en.arb

```diff
@@ -1060,5 +1060,23 @@
   "@the_file_could_not_be_renamed": {
     "type": "text",
     "placeholders": {}
+  },
+  "are_you_sure_you_want_to_delete_member": "Are you sure you want to delete {memberName} from {workspaceName}?",
+  "@are_you_sure_you_want_to_delete_member": {
+    "type": "text",
+    "placeholders": {
+      "memberName": {},
+      "workspaceName": {}
+    }
+  },
+  "user_could_not_be_removed": "The user could not be removed",
```

Comment:
What message is display when delete member with 404 error response?

Reply:
When the server returns 404 error, it displays toast `The user could not be removed`